Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - Inbound Connections From LAN Allowed.json

```diff
@@ -8,7 +8,7 @@
       }
     }
   ],
-  "description": "Allows inbound connections from the Windows Firewall from the Domain and Private network profiles. Allows services like RDP, file sharing, etc. to work.\n\nNote: This policy may show as not applicable on some Windows 10 devices, but it has in fact been applied successfully. It appears to be a bug in the reporting.",
+  "description": "What is it: Allows inbound connections for the Windows Firewall from the Domain and Private network profiles. Allows services like RDP, file sharing, etc. to work. \n Simeon Notes: This policy may show as not applicable on some Windows 10 devices, but it has in fact been applied successfully due to a bug in reporting.",
```

Comment:
**What does this do?**
Allows inbound connections for the Windows Firewall from the Domain and Private network profiles. This allows services such as Remote Desktop and file sharing to function. 

**Why should you use this?**
If you want to allow inbound network traffic by default on Domain and Private network profiles.

**What is the end-user impact?**
...

**Learn more**
[Internet Explorer security zones registry entries for advanced users](https://support.microsoft.com/en-us/help/182569/internet-explorer-security-zones-registry-entries-for-advanced-users) 

@metamenos should we include this? Jeff is wondering if we should exclude this because it shows that Windows defender firewall is not using the recommended settings to protect your computer